stable/yugabyte/values.yaml

```diff
@@ -70,36 +70,55 @@ serviceEndpoints:
     type: LoadBalancer
     app: "yb-master"
     ports:
-      ui: "7000"
+      http-ui: "7000"
 
   - name: "yb-tserver-service"
     type: LoadBalancer
     app: "yb-tserver"
     ports:
-      yql-port: "9042"
-      yedis-port: "6379"
-      ysql-port: "5433"
+      tcp-yql-port: "9042"
+      tcp-yedis-port: "6379"
+      tcp-ysql-port: "5433"
 
 Services:
   - name: "yb-masters"
     label: "yb-master"
     memory_limit_to_ram_ratio: 0.85
     ports:
-      ui: "7000"
-      rpc-port: "7100"
+      http-ui: "7000"
+      tcp-rpc-port: "7100"
 
   - name: "yb-tservers"
     label: "yb-tserver"
     ports:
-      ui: "9000"
-      rpc-port: "7100"
-      yql-port: "9042"
-      yedis-port: "6379"
-      ysql-port: "5433"
-      ycql-metrics: "12000"
-      yedis-metrics: "11000"
-      ysql-metrics: "13000"
+      http-ui: "9000"
+      tcp-rpc-port: "9100"
+      tcp-yql-port: "9042"
+      tcp-yedis-port: "6379"
+      tcp-ysql-port: "5433"
+      http-ycql-met: "12000"
+      http-yedis-met: "11000"
+      http-ysql-met: "13000"
+
+master: {}
+  ## Value of --rpc_bind_addresses flag of yb-master. The value needs
+  ## to be "0.0.0.0:7100" when using Istio.  ref:
+  ## https://docs.yugabyte.com/latest/reference/configuration/yb-master/#rpc-bind-addresses
+  ##
+  # rpc_bind_addresses: "0.0.0.0:7100"
+
+tserver: {}
```

Comment:
we should be able to get by with a single variable here `bind_address` instead of separate ones for master and tserver.